app/presenters/idv/by_mail/letter_enqueued_presenter.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Idv
+  module ByMail
+    class LetterEnqueuedPresenter
+      include ActionView::Helpers::TranslationHelper
+      include Rails.application.routes.url_helpers
+
+      attr_reader :url_options
+
+      def initialize(idv_session:, current_user:, user_session:, url_options:)
+        @idv_session = idv_session
+        @current_user = current_user
+        @user_session = user_session
+        @url_options = url_options
+      end
+
+      def address_lines
+        [
+          pii.address1,
+          pii.address2,
+          "#{pii.city}, #{pii.state} #{pii.zipcode}",
+        ].compact
+      end
+
+      def button_text
+        if sp
+          t('idv.cancel.actions.exit', app_name: APP_NAME)
+        else
+          t('idv.buttons.continue_plain')
+        end
+      end
+
+      def button_destination
+        if sp
+          return_to_sp_cancel_path(step: :verify_address, location: :come_back_later)
+        else
+          account_path
+        end
+      end
+
+      private
+
+      attr_accessor :idv_session, :user_session, :current_user
+
+      def sp
+        @sp ||= idv_session.service_provider
+      end
+
+      def pii
+        @pii ||= pii_from_idv_session ||
```

Comment:
In the verify info controllers for both in-person and remote we copy the `pii_from_doc` and `pii_from_user` into the `applicant` on the idv session. That is intended to be the source of truth for the attributes from that point forward. `pii_from_doc` and `pii_from_user` are left behind to support some edge cases that appear with the back button.

We should be using `idv_session.applicant || pii_from_gpo_pending_profile`.

Reply:
Changed.